### PR TITLE
lps-95688

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:project-templates

File: modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/bnd.bnd [hunk 1]
```diff
@@ -5,7 +5,7 @@ Export-Package: ${package}.constants
 
 -includeresource:\
 	@com.liferay.util.bridges-[0-9]*.jar!/com/liferay/util/bridges/freemarker/FreeMarkerPortlet.class,\
-	@com.liferay.util.taglib-[0-9]*.jar!/META-INF/*.tld#if (${buildType} != "gradle")
+	@com.liferay.util.taglib-[0-9\.]*.jar!/META-INF/*.tld
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/bnd.bnd]: Hey @lawrence-lee  you removed the #if {} statement here.  Was that intentional?

Comment (gamerson) [reply modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/bnd.bnd]: Nevermind!  I see you did it in a followup commit.

**Comment by petershin** [comment]
Sent to Brian: https://github.com/brianchandotcom/liferay-portal/pull/73622

**Pull request closed without merging**